python/src/aac/io/parser/_parse_source.py

```diff
@@ -37,7 +37,7 @@ def parse(source: str, source_uri: Optional[str] = None) -> list[Definition]:
 
     if linesep not in source:
         sanitized_source = sanitize_filesystem_path(source)
-        if path.lexists(sanitized_source):
+        if path.exists(sanitized_source):
```

Comment:
Why did you change this?

Reply:
This was a change that was made during troubleshooting, when there was a suspected issue with the parser. 

@Coffee2Bits , do you think this could be changed back? since we seemed to have found the issue root to be related to the `io.py helper`

Reply:
Yes, I would change it back to `lexists`. The change during our debug session was just a quick check to make sure that there wasn't a filesystem issue that might.

Reply:
Alright I will add it back

Reply:
Alright the changes have been reverted.